src/leiningen/new/re_frame/project.clj

```diff
@@ -35,21 +36,33 @@
                                     "test/js"{{/test?}}{{#garden?}}
                                     "resources/public/css"{{/garden?}}]
 
-{{#garden?}}
+ {{#garden?}}
 
-  :garden {:builds [{:id           "screen"
-                     :source-paths ["src/clj"]
-                     :stylesheet   {{name}}.css/screen
-                     :compiler     {:output-to     "resources/public/css/screen.css"
-                                    :pretty-print? true}}]}
-{{/garden?}}{{#less?}}
+ :garden {:builds [{:id           "screen"
+                    :source-paths ["src/clj"]
+                    :stylesheet   {{name}}.css/screen
+                    :compiler     {:output-to     "resources/public/css/screen.css"
+                                   :pretty-print? true}}]}
+ {{/garden?}}{{#less?}}
   :less {:source-paths ["less"]
          :target-path  "resources/public/css"}
-{{/less?}}
-
-  :shell {:commands {"open" {:windows ["cmd" "/c" "start"]
-                             :macosx  "open"
-                             :linux   "xdg-open"}}}
+ {{/less?}{{#scss?}}
```

Comment:
`/less?` closing tag is missing a closing `}`. i.e. should be ` {{/less?}}{{#scss?}}`. Causes `clojure.lang.ExceptionInfo: Unclosed tag: less? at line 49, column 10`.